src/App.tsx

```diff
@@ -1,27 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goodsData, setGoods] = useState<{
+    id: number, name: string, color: string }[]>([]);
```

Comment:
consider to make type or interface Good for better reading and maybe reuse it.

```suggestion
interface Good {
    id: number,
   name: string, 
   color: string
 }
   
export const App: React.FC = () => {
  const [goodsData, setGoods] = useState<Good[]>([]);
```